### DISCUSSION TITLE
Mutli or single site

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7379] [General]
I have 3 sections of my website and i'm unsure whether to run a multi site or just a single site.

I have a normal section for regular pages like about us, contact us etc, then a section where i'm going to display products to purchase and the 3rd section is the blog or articles section.

I want to show my articles (summarys teasers etc) in my products and regular site and info in my regular site and blogs in my purchase. So I want to isolate the content into 3 separate sections yet I want to easily share snippets and teasers between each sections.

So I'm unsure of whether to build a multisite or single site?

**Comment by ryanmitchell** [comment]
I would say thats a single site with multiple collections

**Reply by ottz0** [reply]
Is this because it’s easier to get the collections rather than going cross site? It says in the docs that multi sites can be used for sections?

**Comment by ryanmitchell** [comment]
I would only suggest using multi site for translations or different domains.What you described sounds like one site and multiple collections.On 19 Jan 2023, at 12:16, ottz0 ***@***.***> wrote:﻿
Is this because it’s easier to get the collections rather than going cross site? It says in the docs that multi sites can be used for sections?
